src/components/Header.js

```diff
@@ -31,7 +33,8 @@ class Header extends Component{
                     SEE THE CODE
                   </NavLink>
                 </NavItem>
-                <UncontrolledDropdown inNavbar nav>
+                {/* adds dropdown menu */}
+                {/* <UncontrolledDropdown inNavbar nav>
```

Comment:
I really don't love keeping this code for later. It's easy enough to grab from reactstrap let's take it out.

Reply:
will do